tests/Fixer/ClassNotation/ClassAttributesSeparationFixerTest.php

```diff
@@ -1144,6 +1144,179 @@ public function f2() {}
                 ',
                 ['elements' => ['const' => ClassAttributesSeparationFixer::SPACING_NONE, 'method' => ClassAttributesSeparationFixer::SPACING_ONE]],
             ],
+            [
+                '<?php
+                    class A
+                    {
+                        const FOO = 1;
+                        const BAR = 2;
+
+                        /** @var int */
+                        const BAZ = 3;
+
+                        /** @var int */
+                        const NEW = 4;
+                    }
+                ',
+                '<?php
+                    class A
+                    {
+                        const FOO = 1;
+
+                        const BAR = 2;
+
+                        /** @var int */
```

Comment:
All the new test cases show empty lines are removed when there is no PHPDoc, but they don't show empty lines are added when there is a PHPDoc as all input already has these empty lines. I would remove those from input samples.